src/org/scratchOrgCreate.ts

```diff
@@ -328,3 +338,13 @@ const getSignupTargetLoginUrl = async (): Promise<string | undefined> => {
     // a project isn't required for org:create
   }
 };
+
+async function getCapitalizeRecordTypesConfig(): Promise<boolean | undefined> {
+  const configAgg = await ConfigAggregator.create();
+  const value = configAgg.getInfo('org-capitalize-record-types').value as string | undefined;
```

Comment:
the type of `value` is `AnyJson | undefined`, at runtime we have 2 possible options:
1) a string containing `true` or `false`. **NOTE** this is always a string, not a boolean
2 `undefined`

if it's `undefined` then we want to return that so `SettingsGenerator` knows there's no config var and emits the warning.

if not undefined we cast it to `boolean` and return it.